src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsole.cs

```diff
@@ -94,7 +95,7 @@ public IVsUIShell VsUIShell
             get { return ServiceProvider.GetService<IVsUIShell>(typeof(SVsUIShell)); }
         }
 
-        private IVsStatusbar VsStatusBar
+        private IVsStatusbar GetVsStatusBar
```

Comment:
verbs imply a method, whereas properties are nous. Either keep a noun name, or change the implementation to a method instead of a property.

Reply:
Ok. I made it call async one with JTF. 
But only concern it starts state machine for it.